dist/src/main/java/io/camunda/zeebe/gateway/ActorSchedulerComponent.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway;
+
+import io.camunda.zeebe.gateway.impl.configuration.GatewayCfg;
+import io.camunda.zeebe.scheduler.ActorScheduler;
+import io.camunda.zeebe.shared.ActorClockConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+final class ActorSchedulerComponent {
+
+  private final GatewayCfg cfg;
+  private final ActorClockConfiguration clockConfiguration;
+
+  @Autowired
+  public ActorSchedulerComponent(
+      final GatewayCfg cfg, final ActorClockConfiguration clockConfiguration) {
+    this.cfg = cfg;
+    this.clockConfiguration = clockConfiguration;
+  }
+
+  @Bean("actorScheduler")
+  @Autowired
```

Comment:
:question: I don't get why we need the `@Autowired` here since there's nothing being injected?

Reply:
:+1: Just an oversight while trying out a few variants.